EnglishFilter/sections/foreign.txt

```diff
@@ -2996,6 +2996,8 @@ kava.ee##.kava_ad_chilli
 !------------------------------ Finnish ------------------------------
 !---------------------------------------------------------------------
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/78743
+tennisassa.fi#?#.module__content:has(> div[class] > .ad--container)
```

Comment:
I guess you want to block leftover, but this rule doesn't do it properly.
`tennisassa.fi#$?#.page-content__main > div.module > div.module__content div.ad--container:upward(div.module) { remove: true; }`